demo/text-field-validators-demos.html

```diff
@@ -49,20 +58,46 @@ <h3>Synchronous Custom Validator</h3>
 
     <p>Extend <code>Vaadin.TextFieldElement</code> to create your own custom element,
       then override the <code>checkValidity()</code> method to validate the user’s input.</p>
-
-    <vaadin-demo-snippet id="text-field-validators-demos-synchronous-custom-validator" ignore-ie when-defined="vaadin-text-field">
+    <vaadin-demo-snippet id="text-field-validators-demos-synchronous-custom-validator"
+      ignore-ie when-defined="vaadin-text-field">
       <template preserve-content>
-        <employee-role-text-field label="Position"></employee-role-text-field>
-        Valid options: Architect, Developer or Designer
+        <employee-role-text-field label="ID" minlength="10" maxlength="10"></employee-role-text-field>
+        <p>Valid ID: A 10 digit number that sum of the digits are dividable to 10. For
```

Comment:
Should be rephrased

Reply:
What about this?
Valid ID: Use a 10 digit number. The sum of digits must be dividable by 10. 
For example 1111111111.

Reply:
I thought "dividable" isn't even a word, but after googling it I learned it seems to be ok to use that word in general. Though I would still recommend to use the much more common form "divisible" instead since it doesn't look so strange. And in this context when talking about divisibility of numbers (math) it's actually more correct to use "divisible" anyway since that implies it's "evenly divisible" which is what is meant here.

I think the updated one now is ok, but I would write it like this instead (and maybe add another example of a valid number):
`Valid ID: A 10 digit integer where the sum of all digits is divisible by 10. For example 1111111111 or 1234000000.`